eox_hooks/serializers.py

```diff
@@ -20,7 +19,7 @@ class UserSerializer(serializers.ModelSerializer):
     """
 
     class Meta(object):
-        model = User
+        model = get_openedx_user_model()
```

Comment:
I think that we can just use  django.contrib.auth.get_user_model()

Reply:
Thank you, I changed it